resources/vars_dark.scss

```diff
@@ -0,0 +1,17 @@
+@import "vars_common";
```

Comment:
Nit: use - instead of _ for consistency with other existing scss files.

Reply:
Done.